prov/verbs/src/verbs_msg_ep.c

```diff
@@ -608,7 +613,7 @@ fi_ibv_send_buf_inline(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr,
 	wr->sg_list = &sge;
 	wr->num_sge = 1;
 
-	return fi_ibv_send(ep, wr, NULL);
+	return fi_ibv_send_poll_cq_if_needed(ep, wr);
```

Comment:
Are the above changes supposed to be part of this PR? I don't remember seeing them in the first round of review.

Reply:
Yes, this should not be a part of the PR. This is a part of master already